src/host/alias.cpp

```diff
@@ -1126,16 +1113,8 @@ size_t Alias::s_ReplaceMacros(std::wstring& str,
 // - If we found a matching alias, this will be the processed data
 //   and lineCount is updated to the new number of lines.
 // - If we didn't match and process an alias, return an empty string.
-std::wstring Alias::s_MatchAndCopyAlias(const std::wstring& sourceText,
-                                        const std::wstring& exeName,
-                                        size_t& lineCount)
+std::wstring Alias::s_MatchAndCopyAlias(std::wstring_view sourceText, const std::wstring& exeName, size_t& lineCount)
 {
-    // Copy source text into a local for manipulation.
-    auto sourceCopy = sourceText;
-
-    // Trim trailing \r\n off of sourceCopy if it has one.
-    s_TrimTrailingCrLf(sourceCopy);

```

Comment:
Will the source text no longer have trailing crlf's?

Reply:
I remove the trailing newlines in the cooked read class now. Technically we can also continue to remove the CRLF in alias.cpp - it doesn't really matter. I removed this code, because cooked read also depends on the newlines being removed and so this was redundant.